content/en/cloudcraft/getting-started/diagram-multiple-cloud-accounts.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Diagram Multiple Cloud Accounts
+---
+
+Cloudcraft is a tool designed to help visualize and plan cloud architecture in a seamless and efficient manner. While the current live experience does not support diagrams involving multiple cloud accounts, this guide explains how to achieve this using the legacy experience within Cloudcraft.
+
+Follow the steps below to consolidate multiple cloud accounts into a single cohesive diagram.
+
+## Multiple accounts diagram
+
+### 1. Enable the legacy experience
```

Comment:
If this is the only H2 level, recommend removing it and bumping child header out. Feels repetitive.
```suggestion
## 1. Enable the legacy experience
```